index.d.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  BufferedChangeset,
+  Changeset,
+  ValidatedChangeset
+} from 'validated-changeset/dist';
+
+import {
+  ValidationResult,
+  ValidatorMapFunc,
+  ValidatorAction
+} from 'validated-changeset/dist/types';
+
+export {
+  BufferedChangeset,
+  Changeset,
```

Comment:
I believe this should also be exported as default. So that `import Changeset from 'ember-changeset` works.

Not sure if this is the right thing to export, but it should have a default export.

Reply:
Good idea.  One thing I am trying to figure out is if we need a `declare module` statement for it to actually work...

Reply:
@josemarluedke What do you think of the PR in it's current state?